Classes/TIPSPDFViewControllerProxy.m

```diff
@@ -458,19 +458,19 @@ - (BOOL)pdfViewController:(PSPDFViewController *)pdfController didTapOnAnnotatio
     return processed;
 }
 
-/// controller did show/scrolled to a new page (at least 51% of it is visible)
-- (void)pdfViewController:(PSPDFViewController *)pdfController didShowPageView:(PSPDFPageView *)pageView {
-    if ([[self eventProxy] _hasListeners:@"didShowPage"]) {
-        NSDictionary *eventDict = @{@"page": [NSNumber numberWithInteger:pageView.pageIndex]};
-        [[self eventProxy] fireEvent:@"didShowPage" withObject:eventDict];
+/// controller did begin displaying a new page (at least 51% of it is visible)
+- (void)pdfViewController:(PSPDFViewController *)pdfController willBeginDisplayingPageView:(PSPDFPageView *)pageView forPageAtIndex:(NSInteger)pageIndex {
+    if ([[self eventProxy] _hasListeners:@"willBeginDisplayingPageView"]) {
+        NSDictionary *eventDict = @{@"page": [NSNumber numberWithInteger:pageIndex]};
```

Comment:
Small one: You can use `@{ @"page": @(pageIndex) }` to trim this down.

Reply:
Makes sense! Boxed values make it easier to read. I wanted to maintain consistency with the rest of the file. Addressed in https://github.com/PSPDFKit/Appcelerator-iOS/pull/20/commits/b68f9f501b7d5fda3f923e108806b05e23b890e1